less/home-page.less

```diff
@@ -52,10 +97,25 @@
     border-top: 2px dashed @navigation-border;
     display: flex;
     justify-content: space-between;
+    @media@mobile {
+      flex-direction: column; 
+      align-items: center;
+      
+    }
 
     .destination {
       width: 30%;
       margin-bottom: 30px;
+      @media@mobile {
+        width: 100%;
+      }
```

Comment:
take a look at the design image for mobile on this `destination` section and see how you might want to adjust this content under 500px -- you want to have things centered